pkgs/development/python-modules/netaddr/default.nix

```diff
@@ -12,26 +12,31 @@ buildPythonPackage rec {
   version = "1.3.0";
   pyproject = true;
 
-  disabled = pythonOlder "3.7";
-
   src = fetchPypi {
     inherit pname version;
     hash = "sha256-XDw9mJW1Ubdjd5un23oDSH3B+OOzha+BmvNBrp725Io=";
   };
 
-  nativeBuildInputs = [ setuptools ];
+  # Test suite uses internal packaging._musllinux module to detect libc flavor. The module assumes
+  # the python executable is dynamically linked - it then attempts to parse linked library name to
+  # detect musl. It won't work on a static build.
+  postPatch = lib.optionalString (stdenv.targetPlatform.isMusl && stdenv.targetPlatform.isStatic) ''
```

Comment:
I wonder if I should use `if XX else null` here instead of `optionalString` to avoid rebuilds?